tests/e2e/utils/fs.ts

```diff
@@ -41,6 +42,19 @@ export function deleteFile(path: string) {
 }
 
 
+export function rimRaf(path: string) {
```

Comment:
These names are SO close, maybe import via: `import * as rimrafPackage from 'rimraf';`? Then rename the exported function `rimraf`.